src/Parsers/ZetaSQL/ParserZetaSQLQuery.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Parsers/IAST_fwd.h"
+#include "Parsers/Lexer.h"
+#include "Parsers/ZetaSQL/ASTZetaSQLQuery.h"
+#include "Parsers/ZetaSQL/ZetaSQLTranslator.h"
+#include "Parsers/ZetaSQL/ParserZetaSQLQuery.h"
+#include "Parsers/ZetaSQL/ParserFromStatement.h"
+#include "Parsers/ZetaSQL/ParserPipe.h"
+
+namespace DB::ZetaSQL {
```

Comment:
(Cosmetic: Let's use namespace `DB` instead of `DB::ZetaSQL` - the ClickHouse codebase generally avoids nested namespaces.)